source/_posts/2016-05-11-deploying-with-capistrano-3.markdown

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: "Deploying with capistrano 3"
+date: 2016-05-20 08:42:17 +0200
+comments: true
+author: Ivica Lakatoš
+categories:
+  - Ruby and Rails
+tags:
+  - ruby
+  - rails
+  - deployment
+  - capistrano 3
+  - capistrano-rails
+  - capistrano-rbenv
+published: true
+---
+
+In the lifetime of every application the time comes to be presented    to everyone. That's why we have to put our application on a special server which is designed for this purpose. In one word, we need to **deploy** our application. In this post you will see how to deploy app with [Capistrano 3](http://www.capistranorb.com/).  
+
+Capistrano is a great developers tool that is used to automatically deploy projects to remote server.
+
+### Add Capistrano to Rails app
+
+I will assume you already have server set up and an application ready to be deployed remotely.
```

Comment:
I think you are missing the "a" article before server, so it should be "I will assume you already have a server set up"